src/targets/services/installSuggestion.js

```diff
@@ -0,0 +1,42 @@
+import fetch from 'node-fetch'
+import CozyClient from 'cozy-client'
+
+global.fetch = fetch
+
+const schema = {
+  suggestion: {
+    doctype: 'io.cozy.suggestions'
+  },
+  konnectors: {
+    doctype: 'io.cozy.konnectors'
+  }
+}
+
+let suggestion = {}
+try {
+  suggestion = JSON.parse(process.env.COZY_COUCH_DOC)
+} catch (e) {
+  throw new Error(`Wrong formatted suggestion doc: ${e}`)
+}
+
+const client = new CozyClient({
+  uri: process.env.COZY_URL.trim(),
+  schema,
+  token: process.env.COZY_CREDENTIALS.trim()
+}).getStackClient()
+
+async function install(suggestion) {
+  if (suggestion.silenced) return // silenced, so not concerned
+  const slug = suggestion.slug
+  try {
+    await client.fetchJSON(
+      'POST',
+      `/konnectors/${slug}?Source=registry://${slug}/stable`
```

Comment:
FMI, is the source not default ?

Reply:
Good question! I don't know :/
@maximemenager do we have a default source in the stack to install apps/konnectors?

Reply:
Yep, if there is no `Source` parameter, the default `Source` the app installer will use the last stable version from the registry